content/en/security/application_security/_index.md

```diff
@@ -57,6 +63,12 @@ To start configuring your environment to detect and protect threats with ASM, fo
 
 In the [Security Signals Explorer][6], click on any security signal to see what happened and the suggested steps to mitigate the attack. In the same panel, view traces with their correlated attack flow and request information to gain further context.
 
+## Investigate risk introduced in upstream open source libraries and dependencies
+
+<div class="alert alert-info">Risk Management is in beta</a></div>
+
+[ASM Risk Management][] shows you when your services are at risk because they use or have dependencies on open source libraries that have known vulnerabilities. Investigate vulnerability findings and secure your software by following remediation advice or researching the cause of the vulnerability.
```

Comment:
```suggestion
[ASM Risk Management][8] shows you when your services are at risk because they use or have dependencies on open source libraries that have known vulnerabilities. Investigate vulnerability findings and secure your software by following remediation advice or researching the cause of the vulnerability.
```